ports/flatcc/portfile.cmake

```
@@ -0,0 +1,34 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
```

It looks like this can be built dynamically https://github.com/dvidelabs/flatcc/blob/master/CMakeLists.txt#L61

---

It can and we would actually support this with `-DBUILD_SHARED_LIBS=${FLATCC_DYNAMIC}`

However, upstream doesn't export any symbols by default and I was taught not to use `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS`

https://github.com/microsoft/vcpkg/pull/43721#discussion_r1973003701

---

So it is a windows limitation.
```suggestion
if(VCPKG_TARGET_IS_WINDOWS)
    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
endif()
```

---

Made a docs PR: https://github.com/microsoft/vcpkg-docs/pull/456

---

is this a windows limitation or a msvc limitation (in which case would `if(MSVC)` work in portfiles)?